filament/backend/src/Platform.cpp

```diff
@@ -35,7 +33,7 @@
         #include "vulkan/PlatformVkCocoaTouch.h"
     #endif
 #elif defined(__APPLE__)
-    #ifndef FILAMENT_USE_EXTERNAL_GLES3
+    #if !defined(FILAMENT_USE_EXTERNAL_GLES3) && !defined(FILAMENT_USE_SWIFTSHADER)
```

Comment:
Why is this change is only necessary for APPLE platforms?

Reply:
Excellent catch, added this for all desktop platforms.